cmd/easeprobe/main.go

```diff
@@ -54,6 +56,21 @@ func main() {
 		os.Exit(-1)
 	}
 
+	// Create the pid file if the file name is not empty
+	if len(strings.TrimSpace(c.Settings.PIDFile)) > 0 {
+		d, err := daemon.NewPIDFile(c.Settings.PIDFile)
+		if err != nil {
+			log.Fatalf("Fatal: Cannot create the PID file: %s!", err)
+			os.Exit(-1)
+		}
+		log.Infof("Successfully create the PID file: %s", d.PIDFile)
+		defer d.RemovePIDFile()
+	} else {
+		log.Info("No PID file is created.")
```

Comment:
```suggestion
		log.Info("Skipping PID file creation (pidfile empty).")
```

Reply:
I couldnt find a better way to word this, but i think its good if we mention the reason somehow (could be an AHA! moment for someone)